externs/shaka/text.js

```diff
@@ -451,6 +452,9 @@ shaka.extern.TextParser = class {
  *     The absolute start time of the segment in seconds.
  * @property {number} segmentEnd
  *     The absolute end time of the segment in seconds.
+ * @property {number} offsetStart
```

Comment:
Since this is specific to VTT, let's call it something like `vttOffset`.  That will make it clear to other parsers not to use it.